docs/extensibility/ux-guidelines/per-monitor-awareness-extenders.md

```diff
@@ -101,13 +103,18 @@ Whenever inside mixed-mode DPI scenarios (e.g. different UI elements rendering i
 #### Out-of-process UI
 Some UI is created out-of-process and if the creating external process is in a different DPI awareness mode than Visual Studio, this can introduce any of the previous rendering issues.
 
-#### Windows Forms controls, images, or windows not displaying
+#### Windows Forms controls, images or layouts rendered incorrectly
+Not all of the Windows Forms content support PMA mode. As a result, you may see rendering issue with incorrect layouts or scaling. A possible solution here is,  force Windows Forms content to be rendered in "System Aware"  DpiAwarenessContext ( refer to [Forcing a control into a specific DpiAwarenessContext](https://github.com/MicrosoftDocs/visualstudio-docs-pr/blob/master/docs/extensibility/ux-guidelines/per-monitor-awareness-extenders.md#forcing-a-control-into-a-specific-dpiawarenesscontext)).
```

Comment:
"visualstudio-docs-pr" isn't a public repo, so non-MS people won't be able to see this file